### PR TITLE
Use console for install log

### PR DESCRIPTION
**Pull request by DecoyRS**

### CODE REVIEW

File: protocol/build.gradle.kts [hunk 1]
```diff
@@ -49,6 +49,7 @@ tasks {
         val cppLibraryOutput = File(cppOutputRoot, "Library")
         val ktLibraryOutput = File(ktOutputRoot, "Library")
 
+        inputs.dir(modelDir.resolve("lib").resolve("ue4"))
```
Comment (LeoMash) [inline protocol/build.gradle.kts:52]: As already discussed it's not enough to depend only on input model files. Generation task should also be invalidated on rd-gen version change

**Review by LeoMash — CHANGES_REQUESTED** [review]

File: protocol/build.gradle.kts (hunk 1 above)
Comment (DecoyRS) [reply protocol/build.gradle.kts:52]: is it though? As we've discussed with Ivan M, changing rdgen version will invalidate whole gradle build description and *might trigger generation tasks, need to check it in the first place

Comment (LeoMash) [reply protocol/build.gradle.kts:52]: Please remember that we have two rd-gen-s, one used at build time and changing its version would probably invalidate gradle description and second one coming from sdk snapshot. And I doubt that new rd-gen coming from there will invalidate something. But gradle is magic, so testing is required

Comment (DecoyRS) [reply protocol/build.gradle.kts:52]: I'll check the next time snapshot gets updated if it'll trigger regeneration of models. Until then, let's keep this review on hold

**Review by LeoMash — APPROVED** [review]

**Pull request merged**